### DISCUSSION TITLE
Want to server generated js/chunks/css file from CDN

### DISCUSSION THREAD

**Discussion by pandey-adarsh147** [sveltejs/kit#7254] [Q&A]
I am using Node adapter, I can see client/_app directory is generated from npm run build. I want to serve these file from CDN. Not able to find way to modify URL in html file or js file. 

Can anyone help here?

**Comment by wwwdepot** [comment]
See https://kit.svelte.dev/docs/configuration#paths

`svelte.config.js` example:

```js
/** @type {import('@sveltejs/kit').Config} */
const config = {
	kit: {
		paths: {
			assets: "https://cdn.example.com"
		}
	}
};
```
